client/app/scripts/superdesk/services/modalService.js

```diff
@@ -43,5 +43,53 @@ define(['angular'], function(angular) {
                 return delay.promise;
             };
 
+        }])
+        .directive('sdModal', ['$document', function($document) {
+
+            return {
+                template: [
+                    '<div class="modal fade hide">',
+                    '<div class="modal-dialog" ng-if="isModalOpen"><div class="modal-content" ng-transclude></div></div>',
+                    '</div>'].join(''),
+                transclude: true,
+                link: function(scope, element, attrs) {
+
+                    var content, _initialized = false;
+                    var trigger = attrs.open;
```

Comment:
this should be fine because it does transclude so this limitation will be only for this modals template.